tools/build_framework.sh

```diff
@@ -25,9 +25,9 @@ echo ARTIFACT_FILES=$ARTIFACT_FILES
 CLI_DIR=${CLI_DIR:=${FRAMEWORK_DIR}/cli}
 UNIVERSE_DIR=${UNIVERSE_DIR:=${FRAMEWORK_DIR}/universe}
 CLI_EXE_NAME=${CLI_EXE_NAME:=dcos-${FRAMEWORK_NAME}}
+BUILD_BOOTSTRAP=${BUILD_BOOTSTRAP:=yes}
 
-TOOLS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-REPO_ROOT_DIR=$(dirname $TOOLS_DIR) # note: need an absolute path for REPO_CLI_RELATIVE_PATH below
+source $TOOLS_DIR/init_paths.sh
```

Comment:
The $TOOLS_DIR requirement on calling scripts can be removed by just doing the following:
```
TOOLS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
source $TOOLS_DIR/init_paths.sh
```